std/python/lib/threading/Thread.hx

```diff
@@ -21,6 +21,23 @@
  */
 package python.lib.threading;
 
+typedef ThreadOptions = {
+	@:optional var group : Dynamic;
+	@:optional var target : Dynamic;
+	@:optional var name : String;
+	@:optional var args : Tuple<Dynamic>;
+	@:optional var kwargs : KwArgs<Dynamic>;
```

Comment:
Not sure `KwArgs` makes any sense here. Should probably be just `python.Dict<String,Dynamic>`.